Couldn't import script, path must be string

## Actual Behavior
1. Start debugging the app
2. Receive following error: `[vscode-react-native] [Warning] Couldn't import script at <http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=false&minify=false>. Debugging won't work: Try reloading the JS from inside the app, or Reconnect the VS Code debugger: path must be a string`

## Expected Behavior
1. Start debugging the app
2. Works 

## Software versions
- React-Native VS Code extension version: 0.2.4
- VSCode version: 1.8.1
- OS platform and version: OSX Sierra 10.12.2
- NodeJS version: 7.3.0
- React Native version: 0.38.1

## Outputs (Include if relevant)
- Output of the Debug Console (View -> Toggle Debug Console):
```
vscode-react-native] [Warning] Couldn't import script at <http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=false&minify=false>. Debugging won't work: Try reloading the JS from inside the app, or Reconnect the VS Code debugger: path must be a string
```

This work correctly on my windows machine, although I don't have the same nodeJS and VSCode versions. Running the command in the terminal work perfectly BTW. 

Thanks

Sorry, I was wondering where err was defined. 
This is the updated info:
```
[vscode-react-native] [Warning] Couldn't import script at <http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=false&minify=false>. Debugging won't work: Try reloading the JS from inside the app, or Reconnect the VS Code debugger: path must be a string
[vscode-react-native] "AssertionError: path must be a string\n    at Module.require (module.js:497:3)\n    at require (/Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/out/debugger/appWorker.js:82:69)\n    at /Users/dev/code/AwesomeProject/.vscode/.react/index.android.bundle:72599:2\n    at ContextifyScript.Script.runInContext (vm.js:37:29)\n    at Object.exports.runInContext (vm.js:69:17)\n    at /Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/out/debugger/appWorker.js:97:16\n    at _fulfilled (/Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/node_modules/q/q.js:834:54)\n    at Promise.then.Q.nextTick.self.promiseDispatch.done (/Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/node_modules/q/q.js:863:30)\n    at Promise.__dirname.Promise.promise.promiseDispatch (/Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/node_modules/q/q.js:796:13)\n    at /Users/dev/.vscode/extensions/vsmobile.vscode-react-native-0.2.4/node_modules/q/q.js:857:14"
```

I'll try to look at the files mentionned.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+## 0.2.5
+* Removed `require` from app worker context to work around change in node 7. `__debug__.require` is still present for applications which need it
+
 ## 0.2.4
 * Allowed using this extension in a workspace with the react-native project in a subfolder.
 * Ignore references to inline sourcemaps, in the hopes of finding another reference to a map file

--- a/src/debugger/appWorker.ts
+++ b/src/debugger/appWorker.ts
@@ -26,7 +26,6 @@ interface DebuggerWorkerSandbox {
     __dirname: string;
     self: DebuggerWorkerSandbox;
     console: typeof console;
-    require: (id: string) => any;
     importScripts: (url: string) => void;
     postMessage: (object: any) => void;
     onmessage: (object: RNAppMessage) => void;
@@ -135,7 +134,6 @@ export class SandboxedAppWorker {
             __dirname: path.dirname(scriptToRunPath),
             self: null,
             console: console,
-            require: (filePath: string) => scriptToRunModule.require(filePath), // Give the sandbox access to require("<filePath>");
             importScripts: (url: string) => this.importScripts(url), // Import script like using <script/>
             postMessage: (object: any) => this.gotResponseFromDebuggerWorker(object), // Post message back to the UI thread
             onmessage: null,

